From edfbff7a23dd93acdf86261cb71ac72f634d846c Mon Sep 17 00:00:00 2001
From: Dmitry Rozhkov <dmitry.rozhkov@jollamobile.com>
Date: Fri, 24 Aug 2012 15:04:46 +0300
Subject: [PATCH] Don't build upstart plugin

---
 plugins/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 5e48133..d6e8c0e 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -4,7 +4,6 @@ SUBDIRS = 	     \
 	gconf        \
 	cgroups      \
 	delay        \
-	upstart      \
 	auth         \
 	dbus         \
 	dbus-signal
-- 
1.7.9.5

